docs/source/java/install.rst

```diff
@@ -143,8 +143,31 @@ Installing Nightly Packages
 Arrow nightly builds are posted on the mailing list at `builds@arrow.apache.org`_.
 The artifacts are uploaded to GitHub. For example, for 2022/03/01, they can be found at `Github Nightly`_.
 
-Maven cannot directly use the artifacts from GitHub.
-Instead, install them to the local Maven repository:
+Installing from Apache Nightlies
+********************************
+
+Steps involved to install to local Maven repository using Apache Nightlies Repository:
+
+1. Is needed to add Apache Nightlies Repository to the Maven/Gradle project. For Maven pom.xml is needed to add arrow/java directory.
+
+.. code-block:: xml
+
+    <properties>
+        <arrow.version>9.0.0.dev191</arrow.version>
```

Comment:
Does the nightlies directory contain only 1 nightly build? Shouldn't it contain multiple nightly builds? I think it was mentioned to keep around a few weeks of them.

In such case, how does the user know which version to put here? We might want to write somewhere that you are expected to go to https://nightlies.apache.org/arrow/java/org/apache/arrow/arrow-memory/ (for example) and see which versions are vailable, or have some other way to know which is the right version.

Reply:
This is an incremental directory, it means we haver more Java artifacts as more nightly build do we have. Pending to define what could the maximum versions storage.

Adding more detail on the steps.